README.md

```diff
@@ -89,7 +89,7 @@ For enhanced file previews (with `scope.sh`):
 
 * `img2txt` (from `caca-utils`) for ASCII-art image previews
 * `w3mimgdisplay`, `ueberzug`, `mpv`, `iTerm2`, `kitty`, `terminology` or `urxvt` for image previews
-* `convert` (from `imagemagick`) to auto-rotate images and for SVG previews
+* `convert` (from `imagemagick`) to auto-rotate images and for image previews
```

Comment:
This was intentionally SVG so I'm not sure why this change is in this PR?
It's a happy coincidence though because I recently merged a PR replacing it with rsvg-convert.
So let's leave it in.